prediction/src/algorithms/segment/src/data_generation.py

```diff
@@ -0,0 +1,63 @@
+import glob
+import os
+
+import numpy as np
+import pylidc as pl
+
+
+def get_dicom_paths(in_docker=True):
+    """Return DICOM paths to all LIDC directories
+    e.g. ['../images_full/LIDC-IDRI-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178/' \
+          '1.3.6.1.4.1.14519.5.2.1.6279.6001.179049373636438705059720603192']"""
+    if in_docker:
+        return glob.glob("../images_full/LIDC-IDRI-*/**/**")
+    else:
+        return glob.glob("../tests/assets/test_image_data/full/LIDC-IDRI-*/**/**")
+
+
+def prepare_training_data():
+    INTERMEDIATE_MALICIOUS = 3
+
+    current_dir = os.path.dirname(os.path.realpath(__file__))
+    assets_dir = os.path.abspath(os.path.join(current_dir, '../assets'))
```

Comment:
how about something like this?

```python
...
parent_dir = os.path.dirname(current_dir)
assets_dir = os.path.abspath(os.path.join(parent_dir, 'assets'))
```

Reply:
there's also 2 other times this same code block is used... you could consider making it a top level constant.

Reply:
added `get_assets_dir()` in 28660bf681dad4acae8e20ee15433d8a48884384